api/models/batch_job.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import "github.com/gojek/merlin/mlp"
+
+type BatchJob struct {
```

Comment:
I think we shouldn't put `BatchJob` and `EndpointJob` under `models` package. It's better to put it closer to the `work` package